lib/src/util/validation_util.dart

```diff
@@ -44,6 +50,15 @@ class ValidationUtil {
       }
 
       window.console.warn('VALIDATION WARNING: $message');
+
+      if (component != null) {
+          window.console.groupCollapsed('(Warning info)');
+
+          window.console.log(component);
+          window.console.log('props: ${component.props}');
```

Comment:
`UiProps.toString` has pretty-printing built in, so this will pretty-print them.